request.py

```diff
@@ -15,6 +15,7 @@ def __init__(
             title (str): The title of the request (should contain the role if 'role' is None).
             end_time (int): The end time of the request as a timestamp.
             role (str, optional): The role being requested. Defaults to None.
+            num_users: The number of users that cast a vote in that thread.
```

Comment:
Docstring doesn't match the actual init arguments. You're right though, that it always starts at 0 so we don't need to pass it in to the constructor, so you can remove this line.